### PR TITLE
#335  Render options on product page

### LINKED ISSUE #335: tests_views.py:662: Render options on product page
The puzzle `315-9094883c` from #315 has to be resolved: 

https://github.com/fidals/stroyprombeton/blob/ffac41ab668aeb261949dc20355c27740c169204/stroyprombeton/tests/tests_views.py#L662-L662

The puzzle was created by duker33 on 14-Nov-18. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by duker33**
Closes #335

### CODE REVIEW

**Comment by artemiy312** [comment]
It seems CI hangs out. I'll leave one of drone agents to fix this

File: templates/catalog/product.html [hunk 1]
```diff
@@ -41,81 +41,86 @@
...
+                </td>
+
+                <td class="option-td">
+                  {% include 'catalog/count_input.html' %}
+                </td>
+                <td class="option-td">
+                  <span class="js-option-sum">{{ option.price|format_price }}</span>
+                </td>
+                <td class="option-td">
+                  {# @todo #335:30m  Adapt order button to Options. #}
```
Comment (artemiy312) [inline templates/catalog/product.html]: What exactly should we do with the button? Or is it part of the issue to find out this?

Comment (duker33) [reply templates/catalog/product.html]: i'll provide more description

**Comment by duker33** [comment]
> It seems CI hangs out. I'll leave one of drone agents to fix this

@artemiy312 , it seems agents worked fine. Timeout was just code issue. We see here strange behavior with waiting 43 mins because of only one timeout. But one agent has this issue too.

**Pull request merged**